README.md

```diff
@@ -1 +1,176 @@
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://guides.rubyonrails.org/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **[Models]**
+- **[Controllers]**
+- **[Views]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ gem install rails
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git git@github.com:Nessrine88/My_Wallet.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd my-project
+  gem install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  rails server
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  rspec spec/models/model_name_spec.rb
+```
+
+### Deployment
+
+[E-Pocket](https://my-budget-ror-app.onrender.com/users/sign_up)
+<!--
+Example:
+
+```sh
+
+```
+ -->
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+ 
+### Video-Demo
+ [Video-demo](https://www.loom.com/share/eaa70b8b64a04d47b87710f0c0e69372?sid=02832cf5-3cab-467c-a537-88e3187ca61c)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Macherki Nessrine**
+
+- GitHub: [@githubhandle](https://github.com/Nessrine88)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/nessrine-macherki-86959196/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Models]**
+- [ ] **[Authentication with Devise]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Nessrine88/My_Wallet/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a star on GitHub
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for giving me this project to build
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+Special thanks to [Gregoire Vella](https://www.behance.net/gregoirevella) for their exceptional design work on the [Snapscan iOs Design and Branding](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=).
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
```

Comment:
- [ ] Good work on your project! However, it seems that the license file is currently missing. To ensure compliance with open-source standards and make your project more accessible to others, please add a license file to your codebase. Adding a license file helps users understand how they can use, modify, and share your project. You can choose a license that aligns with your project's goals and legal requirements. Common licenses include MIT, Apache, or GPL.
![image](https://github.com/Nessrine88/My-Budget-RoR-app/assets/55840999/04353200-0584-4814-a2ad-7c82288c145e)

Keep up the good work, and let me know if you have any questions or need further assistance! 🚀📝